src/runtime_src/core/edge/skd/sk_daemon.cpp

```diff
@@ -50,6 +52,28 @@ void freeBO(unsigned int boHandle)
   xclFreeBO(devHdl, boHandle);
 }
 
+int logMsg(xrtLogMsgLevel level, const char* tag,
+		       const char* format, ...)
+{
+  static auto verbosity = xrt_core::config::get_verbosity();
+  if (level <= verbosity) {
```

Comment:
Please use _early exit_. Invert the if logic so the core code in not inside indentation.
```
if (level > verbosity)
  return 0
```

Reply:
done